src/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.card__image-container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  color: var(--text-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: var(--text-gray);
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review-text {
+  color: var(--text-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-block {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: var(--text-gray);
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  color: var(--text-color);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 100%;
+  min-height: 40px;
+  padding: 12px 70px;
```

```suggestion
  padding: 12px;
```
no need to set such big horizontal padding
if test will be bigger - no will be place for it